packages/shared/alert/AlertsPanel.tsx

```diff
@@ -120,12 +122,16 @@ const AlertAccordionItem: React.FC<AlertAccordionItemProps> = ({
           <AlertsBody error={!_.isEmpty(loadError)}>
             {loaded &&
               alerts.length > 0 &&
-              alerts.map((alert) => (
-                <AlertItem
-                  key={alert?.rule?.id}
-                  alert={alert as unknown as InternalAlert}
-                />
-              ))}
+              alerts.map((alert) =>
+                AlertItemProp ? (
+                  <AlertItemProp key={alert?.rule?.id} alert={alert} />
+                ) : (
+                  <AlertItem
+                    key={alert?.rule?.id}
+                    alert={alert as unknown as InternalAlert}
+                  />
+                )
+              )}
```

Comment:
```suggestion
                  <AlertItemComponent key={alert?.rule?.id} alert={alert} />           
```
Make AlertItem the default value for `AlertItemComponent` prop.